### PR TITLE
Nitya edits

### PR DESCRIPTION
**Pull request by nitarakad**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -247,10 +247,10 @@ All experiments are done with Python 3.7 with several packages including PyTorch
...
+We used MNIST as our source domain and K-MNIST as our target domain. In other words, our model aims to translate images of Kannada numerals to ones that look like typical Arabic numerals for domain adaptation. 
 
 #### Amount of Supervision
-Assuming that a well-studied source domain dataset is available, we use the complete MNIST dataset with all labels. On the other hand, we do experiments with several possible scenarios regarding the target domain dataset. Specifically, we compare the classification accuracy of the translated images when we use the K-MNIST training set with no labels, partial labels, and all labels.
+Assuming that a well-studied source domain dataset is available, we use the complete MNIST dataset with all labels. We also do experiments with several possible scenarios regarding the target domain dataset. Specifically, we compare the classification accuracy of the translated images when we use the K-MNIST training set with no labels, partial labels, and all labels.
```
Comment (ast0414) [inline docs/index.md]: I think the previous one is better aligned with the context of:
- we always use all labels for MNIST throughout the experiments, but try to see the impact of changes in the number of labels used for K-MNIST.
How do you think?

**Review by ast0414 — COMMENTED** [review]
Thanks for your hard work.
I left one comment for line 253, please review it.
Everything else looks fine.

File: docs/index.md (hunk 1 above)
Comment (nitarakad) [reply docs/index.md]: Youre right! I thought I changed that back. My bad.

Comment (ast0414) [reply docs/index.md]: No problem at all. Thank you for your review!

**Review by ast0414 — APPROVED** [review]

**Pull request merged**